### PR TITLE
[stable10] adjust badges on core readme

### PR DESCRIPTION
**Pull request by phil-davis**
Backport #32399 

The master PR had backport-request on it. I guess we want to keep ``README.md`` reasonably in sync in ``stable10``

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,8 @@
 # ownCloud Core
 
-[![Build Status](https://jenkins.owncloud.org/buildStatus/icon?job=owncloud-core/core/master)](https://jenkins.owncloud.org/job/owncloud-core/job/core/job/master/)
+[![Build Status](https://drone.owncloud.com/api/badges/owncloud/core/status.svg?branch=master)](https://drone.owncloud.com/owncloud/core)
```
Comment (patrickjahns) [inline README.md]: stable10 as branch?

Comment (DeepDiver1975) [reply README.md]: Stable10.

**Comment by phil-davis** [comment]
Build Status badge adjusted to show ``stable10`` status.

**Comment by phil-davis** [comment]
@patrickjahns ready for review again.

**Review by patrickjahns — APPROVED** [review]
LGTM

**Pull request merged**